jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/test/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListPresenterTest.java

```diff
@@ -739,19 +749,6 @@ public void testOpenTaskView() {
                      captor.getAllValues().get(1).getIdentifier());
     }
 
-    @Test
-    public void testCreateDataSetProcessInstanceCallbackOnError() {
-        final ProcessInstanceListPresenter spy = spy(presenter);
-        final ClientRuntimeError error = new ClientRuntimeError("");
```

Comment:
Regarding ClientRuntimeError class is not used, maybe the import could be removed